cub/test/test_util.h

```diff
@@ -367,77 +367,75 @@ std::size_t TotalGlobalMem()
  * Random bits generator
  ******************************************************************************/
 
-int g_num_rand_samples = 0;
-
 template <typename T>
 bool IsNaN(T /* val */)
 {
   return false;
 }
 
 template <>
-__noinline__ bool IsNaN<float>(float val)
+__noinline__ inline bool IsNaN<float>(float val)
```

Comment:
This is really funny :D

Reply:
@elstehle discussed with me if we could drop the `__noinline__` entirely. The original commit introducing it (https://github.com/NVIDIA/cub/commit/5da4c84f25118045cb8b76c9e57660f09423d3e8) does not provide a rational for the change. It's also 9 years old, so could be irrelevant today.